terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -0,0 +1,1261 @@
+      version = "0.6.3"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-06.mgr.suse.de/system"
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//provider "libvirt" {
+//  alias = "overdrive3"
+//  uri = "qemu+tcp://overdrive3.mgr.suse.de/system"
+//}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-50-"
+  use_avahi   = false
+  domain      = "mgr.suse.de"
+  images      = [ "sles12sp4o", "sles12sp5o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "sles15sp5o", "slemicro51-ign", "slemicro52-ign", "slemicro53-ign", "slemicro54-ign", "slemicro55-ign", "almalinux9o", "centos7o", "libertylinux9o", "oraclelinux9o", "rocky8o", "rocky9o", "ubuntu2004o", "ubuntu2204o", "debian11o", "debian12o", "opensuse155o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+    additional_network = "192.168.50.0/24"
+  }
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//module "base_arm" {
+//  providers = {
+//    libvirt = libvirt.overdrive3
+//  }
+//
+//  source = "./modules/base"
+//
+//  cc_username = var.SCC_USER
+//  cc_password = var.SCC_PASSWORD
+//  name_prefix = "suma-bv-50-"
+//  use_avahi   = false
+//  domain      = "mgr.suse.de"
+//  images      = [ "opensuse154armo", "opensuse155armo" ]
+//
+//  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+//  use_mirror_images = true
+//
+//  testsuite = true
+//
+//  provider_settings = {
+//    pool        = "ssd"
+//    bridge      = "br1"
+//  }
+//}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "srv"
+  image              = "slemicro55-ign"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:a1"
+    memory             = 40960
+    vcpu               = 10
+    data_pool          = "ssd"
+  }
+  runtime = "podman"
+  container_repository = "registry.suse.de/devel/galaxy/manager/head/containers/suse/manager/5.0/server"
+  helm_chart_url = "oci://registry.suse.de/devel/galaxy/manager/head/charts/suse/manager/5.0/server-helm"
+
+  server_mounted_mirror          = "minima-mirror-ci-bv.mgr.suse.de"
+  java_debugging                 = false
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
```

Comment:
Does it make sense to keep the update option for container ?

Reply:
Honestly, I am not sure about all of those settings for the containerized server. Maybe @cbosdo knows more?

Reply:
This parameter is used for the host, not the image. Some of those parameters are maybe not even implemented, but just copied from the `server` module. For instance I think `monitored` is not wired yet. It may be as simple as adding a `monitored = var.monitored` in the `modules/server_containerized/main.tf`